### ISSUE TITLE
Update profile user-friendly error messages

### ISSUE THREAD

**Issue by dirceudn** [ReachFive/identity-android-sdk#53]
There is an error when I try update a user profile like this:

```
 val profileAddress = ProfileAddress(streetAddress = address, locality = city,
                postalCode = zipcode, country = country
                , company = "", title = "", raw = "", addressType = "", deliveryNote = "",
                isDefault = true, phoneNumber = phoneNumber, recipient = "", region = "")
        val address = listOf(profileAddress)
        val newProfile = Profile(givenName = firstname, familyName = lastname, addresses = address, birthdate = birthday)
```
```
val token = connector.authToken
        if (token != null) {
            reach5.updateProfile(token, profile = newProfile, success = {
                Timber.tag(TAG).d("Profile update $it")
                attachedView?.onInfoSet()
            }, failure = { error ->
                attachedView?.onLoadCredentialsError(error)
            })
        }
```

`ReachFiveError(message=Technical Error, exception=null, data=ReachFiveApiError(error=server_error, errorDescription=Unexpected error occurred, errorDetails=null))
`

The message is not clear about the error. If has a bad request 400 in the server side or there's something wrong with my profile.  I don't know.

**Comment by roxanemace** [comment]
Hi @dirceudn, indeed it's quite annoying to miss the complete error message. I'll improve the `ReachFiveError` model.

**Comment by roxanemace** [comment]
I've released a [`v5.3.0`](https://github.com/ReachFive/identity-android-sdk/releases/tag/v5.3.0) which improves the ReachFive error model. You'll get now the HTTP code response and the friendly-user message. I hope it will help, otherwise, don't hesitate to reopen the issue!

**Comment by dirceudn** [comment]
Thank you @roxanemace :)
